src/websockets/legacy/compatibility.py

```diff
@@ -22,3 +22,12 @@ def loop_if_py_lt_38(loop: asyncio.AbstractEventLoop) -> Dict[str, Any]:
 
         """
         return {"loop": loop}
+
+
+if sys.version_info[:2] < (3, 11):
+    from .async_timeout import timeout as asyncio_timeout  # pragma: no cover
+else:  # pragma: no cover
```

Comment:
I made changes in 399db68d so you can drop the `# pragma: no cover` on the two lines above.